app/controllers/lives_controller.rb

```diff
@@ -36,6 +36,25 @@ def update
     end
   end
 
+  def destroy
+    if @live.author != current_user
```

Comment:
##### Room for improvement

This check is performed twice; Here and in `lives#edit`: https://github.com/lucascaton/easy-lives/blob/826f6ab93c575a17db25e78097526d61e0d9825c/app/controllers/lives_controller.rb#L14-L16
I decided to maintain the duplication to keep the PR single-purposed, but also because it might be a good starting point for a live about DRY.

Reply:
**Update:** rubocop forced me to do it ... i extracted to another `before_action`